sema/ro/creator/__main__.py

```
@@ -0,0 +1,155 @@
+            "Can be absolute or relative to the specified root."
+        ),
+    )
+    return parser
+
+
+def _find_rocyml(root: str) -> tuple[str, str]:
+    """Find the roc yml file in the root folder"""
+    root = Path(root)
+    rocyml = Path("roc-me.yml")
+    if not root.exists():
+        raise FileNotFoundError(f"root path {root} does not exist")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)